index.html

```diff
@@ -87,6 +92,8 @@ <h2>Registry</h2>
 				<tr>
 					<th><code>entryType</code> Identifier</th>
 					<th>Interface Type(s)</th>
+                                        <th>Available From Timeline</th>
```

Comment:
Should this instead be `<th><a>availableFromTimeline</a></th>` and similarly below?

Reply:
Done.